lib/msf/core/post/windows/priv.rb

```diff
@@ -306,6 +306,7 @@ def capture_lsa_key(bootkey)
       end
 
       rc4 = OpenSSL::Cipher.new("rc4")
+      rc4.decrypt
```

Comment:
This file was the only use of openssl that was being tested and impacted by the update